audio/Cargo.toml

```diff
@@ -18,6 +18,10 @@ servo-media-player = { path = "../player" }
 servo-media-traits = { path = "../traits" }
 smallvec = "0.6.1"
 
+[dependencies.speexdsp]
+git = "https://github.com/rust-av/speexdsp-rs"
```

Comment:
Not https://crates.io/crates/speexdsp ?

Reply:
Ah, it's not released yet.

Reply:
Looks like they're willing to make one soon! https://github.com/rust-av/speexdsp-rs/issues/10